fixtures/accelerate_fsdp_defaults.yaml

```diff
@@ -14,7 +14,7 @@ fsdp_config:
   fsdp_auto_wrap_policy: TRANSFORMER_BASED_WRAP
 
   # this controls the FSDP pipelining
-  fsdp_backward_prefetch_policy: BACKWARD_PRE # set to BACKWARD_PRE for the most time-efficient pipeline
+  fsdp_backward_prefetch: BACKWARD_PRE # set to BACKWARD_PRE for the most time-efficient pipeline
```

Comment:
```suggestion
  fsdp_backward_prefetch: BACKWARD_PRE # set to BACKWARD_PRE for the most time-efficient pipeline
  fsdp_backward_prefetch_policy: BACKWARD_PRE # keep for backward compatiblity for accelerate < 1.0
```

Reply:
I think it works with both keys. I have tried on my side. Maybe you can also confirm

Reply:
This is a good solution, thanks Fabian! Can confirm tuning works on accelerate version 1.0.0 with both.